Makefile

```diff
@@ -24,6 +24,7 @@ build: ${SERVER_BIN}
 
 .PHONY: build-cli
 build-cli: ${CLI_BIN}
+	sudo sh -c 'cp ${CLI_BIN} /usr/local/bin/solver && chmod 755 /usr/local/bin/solver'
```

Comment:
maybe this is just me but I feel like this probablyyyy shouldn't be using sudo under the hood. imo we should just build the cli and then maybe print out like hey you will have to run these commands to move the solvercli binary into your path.